solidity/test/data/fraud.ts

```diff
@@ -84,7 +87,8 @@ export const nonWitnessSignSingleInputTx: FraudTestData = {
     "63ac6776a914e257eccafbc07c381642ce6e7e55120fb077fbed8804e0250162b1" +
     "75ac68ffffffff01b8240000000000001600148db50eb52063ea9d98b3eac91489" +
     "a90f738986f60000000001000000",
-  sighash: "0x5d09cd07392c7163335b67eacc999491a3794c15b88e2b59094be5c5b157064b",
+  sha256preimage:
+    "0x90d29d1021c2a1a381c0c8a39f5ffda96770e1c2ffed835f822500033841b2e3",
```

Comment:
Wouldn't it be easier to store both `sighash` and `preimageSha256` in test data? We would avoid all `sha256(sha256preimage)` in `Bridge.Frauds.test.ts` and called `data.preimageSha256` or `data.sighash` depending on what is needed.

Reply:
I added the `sighash` field to test data in 8d5368a082a858912c01bb5097a06544f657229d.